android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -1589,4 +1697,33 @@ public void onMediaFilesUpdated(Url pageUrl, PlaylistItem[] playlistItems) {
             }
         }
     }
+
+    @Override
+    protected void onDetachedFromWindow() {
+        super.onDetachedFromWindow();
+        try {
+            FullscreenManager fullscreenManager =
+                    BraveActivity.getBraveActivity().getFullscreenManager();
```

Comment:
the same as I mentioned above about accessing an Activity

Reply:
https://github.com/brave/brave-core/pull/29431#discussion_r2135552600